src/main/java/net/datafaker/Faker.java

```diff
@@ -13,7 +13,8 @@
 import java.util.Locale;
 import java.util.Random;
 
-public class Faker extends BaseFaker implements BaseProviders, SportProviders, FoodProviders, MovieProviders, VideoGameProviders {
+public class Faker extends BaseFaker
```

Comment:
@snuyanzin can we fix this? Maybe set the line length to 200 or so?

Reply:
solved by setting line limit to 200